torch/ao/quantization/qconfig_mapping.py

```diff
@@ -92,6 +93,7 @@ def _get_default_qconfig_mapping(is_qat: bool, backend: str, version: int) -> QC
         .set_object_type(torch.nn.functional.conv_transpose3d, qconfig_transpose) \
         .set_object_type(torch.nn.functional.layer_norm, qconfig_layernorm) \
         .set_object_type(torch.nn.LayerNorm, qconfig_layernorm) \
+        .set_object_type(torch.nn.PReLU, default_quint8_weight_qconfig) \
```

Comment:
do you still need to unsqueeze weight with this configuration?

Reply:
no, you don't.